scheduled-jobs/build/bug-sweeps/sweep/Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def buildlib = load("pipeline-scripts/buildlib.groovy")
+    def commonlib = buildlib.commonlib
+
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '90'
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    commonlib.ocpVersionParam('BUILD_VERSION'),
+                ]
+            ]
+        ]
+    )
+
+    echo "Initializing bug sweep for ${params.BUILD_VERSION}. Sync: #${currentBuild.number}"
+
+    currentBuild.displayName = "${params.BUILD_VERSION} bug sweep"
+
+    stage("Version dumps") {
+        buildlib.elliott "--version"
+        sh "which elliott"
+    }
+
+    stage("kinit") {
```

Comment:
Could these be a single "init" stage?

Really this job is so simple I'm not sure you need stages at all...

Reply:
Good point.